src/styles/blocks/_stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
```

Comment:
Don't name file with '_' at the beginning. '_' is used for utility files, like _variables.scss or _typography.scss. For blocks styles we don't use it, so name it 'stars.scss'